File: java/Manusia.java

public class Manusia {
    private String nama;
    private int umur;

    public Manusia() {
        nama = null;
        umur = 0;
    }

    public Manusia(String nama, int umur) {
        this.nama = nama;
        this.umur = umur;
    }

    public Manusia(int umur, String nama) {
        this.umur = umur;
        this.nama = nama;
    }

    public void setUmur(int umur) {
        this.umur = umur;
    }

    public void setUmur(String umur) {
        this.umur = Integer.parseInt(umur);
    }

    @Override
    public String toString() {
        return this.nama + ", " + this.umur + " tahun";
    }
}

File: cpp/Manusia.cpp
#include <iostream>
#include <string>
using namespace std;

class Manusia{

    private:
        string nama;
        int umur;
    
    public:
        Manusia() {
            this->nama = "";
            this->umur = 0;
        }

        Manusia(int umur, string nama) {
            this->nama = nama;
            this->umur = umur;
        }

        Manusia(string nama, int umur) {
            this->nama = nama;
            this->umur = umur;
        }

        void setUmur(int umur) {
            this->umur = umur;
        }

        void setUmur(string umur) {
            this->umur = stoi(umur);
        }

        std::string toString() {
            return this->nama + ", " + std::to_string(this-> umur) + " tahun";
        }

};